pootle/apps/pootle_translationproject/models.py

```diff
@@ -158,7 +158,7 @@ class TranslationProject(models.Model, CachedTreeItem):
 
     language = models.ForeignKey(Language, db_index=True)
     project = models.ForeignKey(Project, db_index=True)
-    real_path = models.FilePathField(editable=False)
+    real_path = models.FilePathField(editable=False, null=True, blank=True)
```

Comment:
well, ive grepped and checked, and run all the pootle_fs tests with treestyle set to None, and there doesnt seem to be any issues.

afaict the relevant bits of code are only ever triggered from Project/TP.save()